sphinx_gherkindoc/cli.py

```diff
@@ -163,24 +163,22 @@ def main() -> None:
     args = parser.parse_args()
 
     if args.version:
-        parser.exit(
-            message="Sphinx (sphinx-gherkindoc) {}".format(sphinx.__display_version__)
-        )
+        parser.exit(message=f"Sphinx (sphinx-gherkindoc) {sphinx.__display_version__}")
 
     set_dry_run(args.dry_run)
     set_verbose(args.verbose)
 
     gherkin_path = pathlib.Path(args.gherkin_path)
     if not gherkin_path.is_dir():
-        parser.error("{} is not a directory.".format(gherkin_path))
+        parser.error(f"{args.gherkin_path} is not a directory.")
```

Comment:
Why the `args.` prefix?  One of the "great joys" I've had with debugging relative path programs is in getting errors like this and not having any clue what the actual path being complained about is...  Probably even happened while working on this very package when I was actively working on it...

Reply:
So the intention here (as with https://github.com/jolly-good-toolbelt/sphinx_gherkindoc/pull/8/files#diff-27df03292cffa651fb903656cef3cde4R181) was to make the error message return to the user what they provided to the value so there isn't confusion over "what is that path? That's not what I typed"

Reply:
Yeah, i get that too, but for a "just convert to f-strings" it feels a little sleezy to sneak that in.